src/theory/theory_inference_manager.h

```diff
@@ -432,6 +435,8 @@ class TheoryInferenceManager
   uint32_t d_numCurrentLemmas;
   /** The number of internal facts added since the last call to reset. */
   uint32_t d_numCurrentFacts;
+  /** Statistics for the inference ids sent via this inference manager. */
+  IntegralHistogramStat<InferenceId> d_inferenceStats;
```

Comment:
I actually think we will want to split this into 3 parts (lemma vs fact vs conflict).  Note that both in strings and in datatypes, there are inferences that may be sent as lemmas or as facts. FYI @yoni206 .